src/app/application.cpp

```diff
@@ -125,6 +125,7 @@ Application::Application(const QString &id, int &argc, char **argv)
     qRegisterMetaType<Log::Msg>("Log::Msg");
 
     setApplicationName("qBittorrent");
+    setOrganizationDomain("qbittorrent.org");
```

Comment:
I suspect this has any real effect on the topic, from Qt doc:
>The value is used by the QSettings class when it is constructed using the empty constructor. This saves having to repeat this information each time a QSettings object is created.

We can still keep it nonetheless, since I didn't found any `QSettings` instance constructed using the empty constructor.